### ISSUE TITLE
Not working for IOS

### ISSUE THREAD

**Issue by GautamV234** [Almoullim/background_location#100]
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    /Users/user/Desktop/test/test/ios/Runner/GeneratedPluginRegistrant.m:10:9: fatal error: module 'background_location' not found
    @import background_location;
     ~~~~~~~^~~~~~~~~~~~~~~~~~~

**Comment by GautamV234** [comment]
Can someone please help with this?

**Comment by manuel-junior** [comment]
I was facing the same problem
Try to remove ios folder and run flutter create .
Make sure to save old folder

**Comment by manuel-junior** [comment]
Are you using mac os?

**Comment by GautamV234** [comment]
Yes I am using Mac OS and want to run the app on the iOS emulator to test it on iOS so deleting the iOS folder wouldn't be the best thing to do.

**Comment by GautamV234** [comment]
> Are you using mac os?

Yes I am using Mac OS and want to run the app on the iOS emulator to test it on iOS so deleting the iOS folder wouldn't be the best thing to do.

**Comment by HugoYG** [comment]
Just run this on your terminal.
 rm ios/Podfile && flutter build ios

**Comment by GautamV234** [comment]
> Just run this on your terminal.
> rm ios/Podfile && flutter build iOS

It is stated that this is needed for running the app on an actual IOS device but I just first want to run the device on an iOS simulator and have gotten the above error while trying to run on an emulator itself. How should I proceed?

**Comment by HugoYG** [comment]
Run the command and then try to run the app on the IOS Simulator u should be fine.

**Comment by Ssavas92** [comment]
Merhava dostum eğer sorunu çözemediysen hala xcode ile "Runner.xcworkspace" dosyasını aç ve pod klasörü altındaki paketlerin ios target seviyelerini 11 e çek düzelecektir.
